code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -4,7 +4,7 @@
 	var/obj/machinery/camera/current = null
 	icon = 'icons/mob/pai.dmi'
 	icon_state = "repairbot"
-	mouse_opacity = 1
+	mouse_opacity = 2

```

Comment:
@AnturK don't fill it with alpha 1 white pixels, when I asked WJ for the bee sprites, he pre-emptively filled the edges with alpha 1 white pixels, and I could see them in game and it was horrible, which is why I removed those pixels and used ```mouse_opacity```